<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1764</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>1611</width>
      <height>931</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="toolTip">
          <string>None</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/pointer.png</normaloff>:/images/pointer.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="toolTip">
          <string>Ellipse</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/ellipse.png</normaloff>:/images/ellipse.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="toolTip">
          <string>ZiGZAG</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/zig.jpg</normaloff>images/zig.jpg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="toolTip">
          <string>Sector</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/sec.png</normaloff>:/images/sec.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="toolTip">
          <string>Brush Color</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/floodfill.png</normaloff>:/images/floodfill.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lineColor">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Line Color</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/linecolor.png</normaloff>:/images/linecolor.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="toFront">
         <property name="toolTip">
          <string>Bring to Front</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/bringtofront.png</normaloff>:/images/bringtofront.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendBack">
         <property name="toolTip">
          <string>Send to Back</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/sendtoback.png</normaloff>:/images/sendtoback.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="flipV">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/flipV.png</normaloff>:/images/flipV.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="flipH">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/flipH.png</normaloff>:/images/flipH.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Opacity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="opacitySlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sliderPosition">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="startSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimum">
            <number>-10</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="sliderPosition">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Line Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="lineBox">
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/line1.png</normaloff>:/images/line1.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/line2.png</normaloff>:/images/line2.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/line3.png</normaloff>:/images/line3.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Zigzag steps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="zigLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Angle Begin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="zigLine_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Angle End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="zigLine_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1764</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
